src/Microsoft.OData.Core/SelectedPropertiesNode.cs

```diff
@@ -831,7 +875,7 @@ private static SelectedPropertiesNode ProcessSubExpand(string nodeName, Selected
         /// <param name="selectList">An enumerable of selected item names.</param>
         /// <param name="expandList">An enumerable of sub expanded nodes.</param>
         /// <returns>The generated SelectedPropertiesNode.</returns>
-        private static SelectedPropertiesNode CombineSelectAndExpandResult(IEnumerable<string> selectList, IEnumerable<SelectedPropertiesNode> expandList)
+        private static SelectedPropertiesNode CombineSelectAndExpandResult(IList<string> selectList, IList<SelectedPropertiesNode> expandList, ODataVersion version)

```

Comment:
>IList<string> [](start = 75, length = 13)

what's the reason to change from "IEnumerable<T>" to "IList<T>"

Reply:
Got reverted because of an outdated fork.